src/dmd/id.d

```diff
@@ -343,6 +343,10 @@ immutable Msgtable[] msgtable =
     { "__switch"},
     { "__switch_error"},
     { "__ArrayCast"},
+    { "HookTraceImpl" },
```

Comment:
DId I miss something in the druntime PR?  Specialized, compiler-recognized symbols should have a `_d_` or `__` prefix. Why is this one different?

Reply:
I added the symbol here to be able to write `Id.HookTraceImpl` instead of `Identifier.idPool("HookTraceImpl")` inside the the remove functions inside `dinterpret.d` and `inline.d`.
While adding that template-function inside druntime I did not think of adding a prefix as this symbol is never accessible unless `core.internal.array.utils` is imported. Should it still have one? or should I use `Identifier.idPool("HookTraceImpl")` instead adding this symbol to `id.d`?

Reply:
Yeah, I would prefer it have a `_d_` prefix.  I'm sorry I didn't pick up on that earlier.

Reply:
If you know the id string ahead of time and it will be used, use the `id.d` approach, its faster and easier to read.

Reply:
https://github.com/dlang/druntime/pull/2686